### PR TITLE
Remove "by income class" from plot titles and also colouring argument

### LINKED ISSUE #92: plot titles should not have "by income class"
Since currently the simulations are being done separately for each income class, the plot titles should not have "by income class". It is more suitable when making a combined CSV of all income classes and then including "by income class" in the title and also colouring by the "income" column.

### PR DESCRIPTION
**Pull request by SaranjeetKaur**
Closes #92 

This pull request includes changes to the `R/health-plots.R` and `tests/testthat/test-health-plots.R` files to modify plot titles by removing references to "by income class". The changes ensure consistency in the plot titles and update corresponding tests to reflect these modifications.

### CODE REVIEW

File: R/health-plots.R [hunk 1]
```diff
@@ -255,18 +254,17 @@ inc_cum <- function(inc,
...
 
   plot_title <- switch(inc,
-    asthma = "Asthma - Cumulative reduction by income class",
-    ckd = "Chronic kidney disease - Cumulative reduction by income class",
-    diabetes = "Diabetes - Cumulative reduction by income class",
-    ischemia = "Ischemic heart disease - Cumulative reduction by income class",
-    stroke = "Stroke - Cumulative reduction by income class"
+    asthma = "Asthma - Cumulative reduction",
+    ckd = "Chronic kidney disease - Cumulative reduction",
+    diabetes = "Diabetes - Cumulative reduction",
```
Comment (jzhu20) [inline R/health-plots.R:259]:
```suggestion
    diabetes = "Diabetes - Cumulative reduction in incidence",
```

File: R/health-plots.R [hunk 2]
```diff
@@ -255,18 +254,17 @@ inc_cum <- function(inc,
...
+    ischemia = "Ischemic heart disease - Cumulative reduction",
```
Comment (jzhu20) [inline R/health-plots.R:260]:
```suggestion
    ischemia = "Ischemic heart disease - Cumulative reduction in incidence",
```

File: R/health-plots.R [hunk 3]
```diff
@@ -255,18 +254,17 @@ inc_cum <- function(inc,
...
+    stroke = "Stroke - Cumulative reduction"
```
Comment (jzhu20) [inline R/health-plots.R:261]:
```suggestion
    stroke = "Stroke - Cumulative reduction in incidence"
```

File: tests/testthat/test-health-plots.R [hunk 4]
```diff
@@ -151,7 +151,7 @@ test_that("inc_cum function works correctly", {
     scale_y_continuous_labels = scales::comma(c(-4769000, -4292000, -3815000, -3338000, -2861000, -2384000, -1908000, -1431000, -954000, -477000, 0))
   )
   expect_s3_class(plot_asthma, "ggplot")
-  expect_equal(plot_asthma$labels$title, "Asthma - Cumulative reduction by income class")
+  expect_equal(plot_asthma$labels$title, "Asthma - Cumulative reduction")
```
Comment (jzhu20) [inline tests/testthat/test-health-plots.R:154]:
```suggestion
  expect_equal(plot_asthma$labels$title, "Asthma - Cumulative reduction in incidence")
```

File: tests/testthat/test-health-plots.R [hunk 5]
```diff
@@ -161,7 +161,7 @@ test_that("inc_cum function works correctly", {
     scale_y_continuous_labels = scales::comma(c(-2098000, -1888000, -1679000, -1469000, -1259000, -1049000, -839000, -629000, -420000, -210000, 0))
   )
   expect_s3_class(plot_ckd, "ggplot")
-  expect_equal(plot_ckd$labels$title, "Chronic kidney disease - Cumulative reduction by income class")
+  expect_equal(plot_ckd$labels$title, "Chronic kidney disease - Cumulative reduction")
```
Comment (jzhu20) [inline tests/testthat/test-health-plots.R:164]:
```suggestion
  expect_equal(plot_ckd$labels$title, "Chronic kidney disease - Cumulative reduction in incidence")
```

File: tests/testthat/test-health-plots.R [hunk 6]
```diff
@@ -171,7 +171,7 @@ test_that("inc_cum function works correctly", {
     scale_y_continuous_labels = scales::comma(c(-4424000, -4084000, -3743000, -3403000, -3063000, -2722000, -2382000, -2042000, -1701000, -1361000, -1021000, -681000, -340000, 0))
   )
   expect_s3_class(plot_db, "ggplot")
-  expect_equal(plot_db$labels$title, "Diabetes - Cumulative reduction by income class")
+  expect_equal(plot_db$labels$title, "Diabetes - Cumulative reduction")
```
Comment (jzhu20) [inline tests/testthat/test-health-plots.R:174]:
```suggestion
  expect_equal(plot_db$labels$title, "Diabetes - Cumulative reduction in incidence")
```

File: tests/testthat/test-health-plots.R [hunk 7]
```diff
@@ -181,7 +181,7 @@ test_that("inc_cum function works correctly", {
     scale_y_continuous_labels = scales::comma(c(-6455000, -5810000, -5164000, -4519000, -3873000, -3228000, -2582000, -1937000, -1291000, -646000, 0))
   )
   expect_s3_class(plot_ihd, "ggplot")
-  expect_equal(plot_ihd$labels$title, "Ischemic heart disease - Cumulative reduction by income class")
+  expect_equal(plot_ihd$labels$title, "Ischemic heart disease - Cumulative reduction")
```
Comment (jzhu20) [inline tests/testthat/test-health-plots.R:184]:
```suggestion
  expect_equal(plot_ihd$labels$title, "Ischemic heart disease - Cumulative reduction in incidence")
```

File: tests/testthat/test-health-plots.R [hunk 8]
```diff
@@ -191,7 +191,7 @@ test_that("inc_cum function works correctly", {
     scale_y_continuous_labels = scales::comma(c(-1198000, -1078000, -959000, -839000, -719000, -599000, -479000, -359000, -240000, -120000, 0))
   )
   expect_s3_class(plot_stroke, "ggplot")
-  expect_equal(plot_stroke$labels$title, "Stroke - Cumulative reduction by income class")
+  expect_equal(plot_stroke$labels$title, "Stroke - Cumulative reduction")
```
Comment (jzhu20) [inline tests/testthat/test-health-plots.R:194]:
```suggestion
  expect_equal(plot_stroke$labels$title, "Stroke - Cumulative reduction in incidence")
```

**Comment by jzhu20** [comment]
Hi @SaranjeetKaur I just made a few suggestions, basically clarifying cumulative reduction in incidence for diseases. Other parts look all good. Thank you.

**Comment by SaranjeetKaur** [comment]
Thanks @jzhu20! I have incorporated your feedback and updated the code accordingly. Merging this PR now.

**Pull request merged**